### PR TITLE
Add support for Tablet Mode's back button

### PR DESCRIPTION
**Pull request by jaigak**
Adds support for using Tablet Mode's back button for navigating between directories.

### CODE REVIEW

File: Files/MultilingualResources/Files.ru-RU.xlf [hunk 1]
```diff
@@ -634,6 +634,10 @@
           <source>Set as lockscreen background</source>
           <target state="translated">Применить как фон экрана блокировки</target>
         </trans-unit>
+        <trans-unit id="BaseLayoutItemContextFlyoutSetAsLockscreenBackground" translate="yes" xml:space="preserve">
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.ru-RU.xlf:637]: Why did you add duplicate string?

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran Can you share a screenshot of how it will look with these changes?

**Comment by Jaiganeshkumaran** [comment]
> @Jaiganeshkumaran Can you share a screenshot of how it will look with these changes?
There is no change in the UI. Only difference is that now the app responds to the tablet mode back button and uses it for navigating to the previous directory instead of switching to the previous app or the Start Screen.

**Review by yair100 — APPROVED** [review]

**Review by lukeblevins — APPROVED** [review]
Thank you!

**Comment by yaichenbaum** [comment]
@Jaiganeshkumaran Will this interfere with the side bar?

**Comment by lukeblevins** [comment]
@yair100 This PR only adds support for the Windows 10 tablet mode back button. It won't affect the UI at all.
